aten/src/ATen/native/EmbeddingBag.cpp

```diff
@@ -86,11 +86,12 @@ void index_select_add<float>(const Tensor &select_indices,
                              const Tensor& offsets,
                              bool include_last_offset) {
   int64_t ddim = src.size(1);
-  auto* src_data = src.data_ptr<float>();
   auto* select_indices_data = select_indices.data_ptr<int64_t>();
   auto* output_data = output.data_ptr<float>();
 
   if (isFastPathIndexSelect(src, output)) {
+    auto src_cnt = src.contiguous();
+    auto* src_data = src_cnt.data_ptr<float>();
```

Comment:
nit: the two lines can be merged into one.
src.contiguous().data_ptr<float>()

Reply:
Makes sense! Thanks!